bundles/binding/org.openhab.binding.ddwrt/src/main/java/org/openhab/binding/ddwrt/ddwrtBindingProvider.java

```diff
@@ -0,0 +1,70 @@
+/**
+ * Copyright (c) 2010-2016 by the respective copyright holders.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.ddwrt;
+
+import org.openhab.core.binding.BindingProvider;
+import org.openhab.core.items.Item;
+
+/**
+ * This interface is implemented by classes that can map openHAB items to
+ * ddwrt binding types.
+ *
+ * Implementing classes should register themselves as a service in order to be
+ * taken into account.
+ *
+ * @author Kai Kreuzer
+ * @author Markus Eckhardt
+ * @since 0.1.0

```

Comment:
Please make all `@author` tags reflect the author(s) of this new code, and make `@since` refer to 1.9.0.